app/assets/web-dev-iframe.html

```diff
@@ -21,7 +21,7 @@
       const styleEl = document.createElement('style')
       styleEl.type = "text/css"
       let styleText = '';
-      if (parent.features && parent.features.chinaUx) {
+      if (new URLSearchParams(window.location.search).get('chinaUx')) {
```

Comment:
Bug here, this will always behave as if we are in china.
`?chinaUx=false` will return the string `'false'` which is truthy.

Instead we can check against `new URLSearchParams(window.location.search).get('chinaUx') === 'true'`